src/platform_impl/ios/ffi.rs

```diff
@@ -359,7 +359,7 @@ pub struct CFRunLoopSourceContext {
     pub perform: Option<extern "C" fn(*mut c_void)>,
 }
 
-pub trait NSString: Sized {
+pub trait NSStringRust: Sized {
```

Comment:
There should probably be a comment here about why it's not just named `NSString` because eventually someone else will come along and want to change it back.